docs/data/pages.ts

```diff
@@ -301,11 +296,18 @@ const pages: MuiPage[] = [
             title: 'DateRangePickerDay',
             plan: 'pro',
           },
-          { pathname: '/x/api/date-pickers/date-time-picker', title: 'DateTimePicker' },
           {
-            pathname: '/x/api/date-pickers/day-calendar-skeleton',
-            title: 'DayCalendarSkeleton',
+            pathname: '/x/api/date-pickers/date-range-picker',
+            title: 'DateRangePicker',
+            plan: 'pro',
+          },
+          { pathname: '/x/api/date-pickers/date-time-field', title: 'DateTimeField' },
+          {
+            pathname: '/x/api/date-pickers/static-date-time-picker-tabs',
```

Comment:
```suggestion
            pathname: '/x/api/date-pickers/date-time-picker-tabs',
```

Reply:
This is a problem once again 🙈 
Is this list auto-generated? Or it was a merge conflict issue?

Reply:
You probably missed this comment? 😢

Reply:
I fixed in the index but not here, my bad...